### PR TITLE
Change echo service default_host to localhost, without port

### PR DESCRIPTION
**Pull request by andreamlin**
Resolves https://github.com/googleapis/gapic-showcase/issues/25

See https://github.com/googleapis/gapic-generator/pull/2346 for gapic-generator CI build using this PR.

### CODE REVIEW

File: schema/echo.proto [hunk 1]
```diff
@@ -36,7 +36,7 @@ option (google.api.metadata) = {
 service Echo {
   // This service is meant to only run locally on the port 7469 (keypad digits
   // for "show").
-  option (google.api.default_host) = "localhost:7469";
+  option (google.api.default_host) = "localhost";
```
Comment (lukesneeringer) [inline schema/echo.proto]: Why this change? We do not have a port in the annotations, so this is needed, yes?

**Comment by andreamlin** [comment]
Closing and opening https://github.com/googleapis/gapic-generator/pull/2350 to actually use the port specified in the service config.
